unfuck-submodules.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIRNAME=$(basename "$PWD")
+if [[ "$DIRNAME" != "hyperlane-monorepo" ]]; then
+  echo "Must be run from the root of the monorepo"
+  exit 1
+fi
+
+DIRS=$(git submodule status | grep -oE "typescript/[a-zA-Z0-9_-]+")
+for DIR in $DIRS ; do
+    echo "Removing '$DIR'"
+    rm -rf "$DIR"
+done
+
+git submodule init
+git submodule update
+
+for DIR in $DIRS ; do
+    echo "Cleaning '$DIR'"
+    pushd "$DIR" || continue
+    rm -rf yarn.lock node_modules
```

Comment:
rm the module's `node_modules` dir shouldn't be necessary I think. Have they created problems for you?

Reply:
🤷‍♀️ 
https://discord.com/channels/935678348330434570/1068617127268990996/1068618656822595675